api/src/apps/hooks.rs

```diff
@@ -83,16 +83,17 @@ impl Apps {
             }
         };
 
-        let statements = match boa::parse(&hook_content, false) {
+        let mut context = Context::default();
+        let statements = match Context::default().parse(&hook_content) {
```

Comment:
You can avoid an extra allocation here.

```suggestion
        let statements = match context.parse(&hook_content) {
```

Reply:
Extra allocation removed